dependencies.yaml

```diff
@@ -60,7 +60,7 @@ libevse-security:
 # OCPP
 libocpp:
   git: https://github.com/EVerest/libocpp.git
-  git_tag: v0.13.0
+  git_tag: feature/656-device-model-initialization-in-cpp
```

Comment:
We usually delete the branches, so the PR in libocpp will be merged first and then we need to update the reference to the commit hash here.

Reply:
Yes so this needs to change before merging indeed. But now you can just checkout this commit and it will build (which is nice to know for the buildserver)